### ISSUE TITLE
Api

### ISSUE THREAD

**Issue by Tom4nt** [Tom4nt/Mega-Soundboard#52]
### Discussed in https://github.com/Tom4nt/Mega-Soundboard/discussions/34

<div type='discussions-op-text'>

<sup>Originally posted by **emanuelfranklyn** June 25, 2022</sup>
It would be great if I could from an external script on my pc using HTTP or web socket or any other kind of method retrieve data about what sound is current playing and receive events once audios start playing or end playing.</div>

**Comment by Noctabou** [comment]
Even play an audio with a request to the API ? I don't really know if there would be a usage for this, since most of programs would play the sound themselves. 
But maybe if the API is available through network, so i could tell my friends Discord i'm coming back using a request to MegaSoundboard to play a file saying "brb" and pressing PTT key. Why not

**Comment by Tom4nt** [comment]
Yes, I think the current idea allows for that. An HTTP API would be available anywhere, depending on your network settings. I actually haven't thought of any use case myself, but I'm going to implement this since it was requested a long time ago. The use case you proposed is the first I've seen for this and it's a good one IMO.
